esrally/mechanic/telemetry.py

```diff
@@ -795,35 +800,52 @@ def attach_to_node(self, node):
     def on_benchmark_start(self):
         if self.process is not None:
             self.process_start = sysstats.process_io_counters(self.process)
+            read_bytes = 0
+            write_bytes = 0
+            io.ensure_dir(self.log_root)
+            log_file = "%s/%s-%s.io" % (self.log_root, self.car.safe_name, self.node.node_name)
+            console.info("%s: Writing start I/O stats to [%s]" % (self.human_name, log_file), logger=self.logger)
```

Comment:
As this is only a temporary file that is necessary due to the way the `start` and `stop` subcommands will work, we should not inform the user about it.